charms/knative-eventing/src/charm.py

```diff
@@ -45,14 +45,13 @@ def _apply_and_set_status(self):
         try:
             self.unit.status = MaintenanceStatus("Configuring/deploying resources")
             self.resource_handler.apply()
-        except (ApiError, ErrorWithStatus) as e:
```

Comment:
What is the motivation for splitting the exception?

Reply:
because later in the code there is a check for instances of errors, that is not the native python implementation